endpoints/openrtb2/auction.go

```diff
@@ -924,6 +941,11 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 	for i := 0; i < len(impIds); i++ {
 		resolvedImp, err := jsonpatch.MergePatch(storedImps[impIds[i]], imps[idIndices[i]])
 		if err != nil {
+			if !json.Valid(imps[idIndices[i]]) {
+				err = errBadIncomingImp
```

Comment:
When the incoming JSON is invalid, it would require a lot of trickery to be able to place the path into the error message.  I did add paths for stored requests and stored imps.  For incoming requests, the error message simply states the incoming request had invalid JSON and then displays the specific error and offset within the request that Unmarshal() produces.